recipe/meta.yaml

```diff
@@ -39,7 +39,6 @@ outputs:
       script:
         - export LIBUNICORN_PATH="${PREFIX}"
         - mkdir source/bindings/python/src
-        - cp source/pkgconfig.mk source/bindings/python/src
         - ${PYTHON} -m pip install source/bindings/python/ -vv
     requirements:
       host:
```

Comment:
Please use  `- {{ pin_subpackage("unicorn", exact=True) }}` here in `host` and build`